etc/config-server.template.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
rammingen uses json5, not json, so I would prefer to avoid `.json` extension. We can use `.conf` instead (`.json5` is also an option).